app/build.gradle

```diff
@@ -1,12 +1,13 @@
 apply plugin: 'com.android.application'
+apply plugin: 'org.jetbrains.kotlin.android'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 32
```

Comment:
Does this not cause a problem in Android 11 and later?

Reply:
The restrictions on file system is applied by Android system by force regardless the target sdk version of libraries